lib/astarte/client/pairing/agent.ex

```diff
@@ -32,6 +44,13 @@ defmodule Astarte.Client.Pairing.Agent do
     end
   end
 
+  @doc """
+  Unregister a device.
+
+  This makes it possible to register it again, even if it already has requested its credentials.
```

Comment:
This allows to register the device again, even if it has already requested its credentials.